Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
-import './App.css';
+import React, { Fragment } from "react";
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import OperationButton from './components/OperationComponents/OperationComponents';
+import "./App.css";
+
+const numPad = [7, 8, 9, 4, 5, 6, 1, 2, 3];
+const operationPad = ["÷", "×", "-", "+", "="];
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <Fragment>
+      <CalculatorDisplay />
+      <div className="bottomContainer">
+        <div className="leftCon">
+          <ActionButton style="actionButton" text="clear" />
+          <div className="numberButtonDiv">
+            {numPad.map(item => {
+              return <NumberButton style="numberButton" text={item} />;
+            })}
+          </div>
+          <ActionButton style="actionButton" text="0" />
+        </div>
+        <div className="operationButtonDiv">
+          {operationPad.map(item => {
+            return <OperationButton style="operationButton" text={item} />;
```

Comment:
If you ever think a JSX element is getting too long with all of its props, you can indent it like so(with each prop. on its own line):
```javascript
return (
  <OperationButton 
    style="operationButton" 
    text={item}
  />
);
```